core/src/test/java/com/github/jsonldjava/core/ContextSerializationTest.java

```diff
@@ -0,0 +1,24 @@
+package com.github.jsonldjava.core;
+
+import com.github.jsonldjava.utils.JsonUtils;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class ContextSerializationTest {
+
+    @Test
+    // Added in order to have some coverage on the serialize method since is not used anywhere.
+    public void serializeTest() throws IOException {
+        final Map<String, Object> json = (Map<String, Object>)JsonUtils
+                .fromInputStream(getClass().getResourceAsStream("/custom/contexttest-0005.jsonld"));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [UI: Usage of GetResource may be unsafe if class is extended (UI_INHERITANCE_UNSAFE_GETRESOURCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#ui-usage-of-getresource-may-be-unsafe-if-class-is-extended-ui-inheritance-unsafe-getresource).
Calling this.getClass().getResource(...) could give results other than expected if this class is extended by a class in another package.